<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stats Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <widget class="QLabel" name="lblEndDate">
          <property name="text">
           <string>Until</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblLength">
          <property name="text">
           <string>Display last N hours</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="chkWorking">
            <property name="text">
             <string>Working</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkIdle">
            <property name="text">
             <string>Idle</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkPaused">
            <property name="text">
             <string>Paused</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkOffline">
            <property name="text">
             <string>Offline</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lblSelectReport">
          <property name="text">
           <string>Select a report</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,2">
          <item>
           <widget class="QCheckBox" name="chkNow">
            <property name="text">
             <string>Now</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="dtEndDate">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="dateTime">
             <datetime>
              <hour>8</hour>
              <minute>0</minute>
              <second>0</second>
              <year>2014</year>
              <month>1</month>
              <day>1</day>
             </datetime>
            </property>
            <property name="minimumDateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>2014</year>
              <month>1</month>
              <day>1</day>
             </datetime>
            </property>
            <property name="minimumDate">
             <date>
              <year>2014</year>
              <month>1</month>
              <day>1</day>
             </date>
            </property>
            <property name="displayFormat">
             <string>dd MMM hh:mm</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cbReport">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>RN Usage</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Job Usage</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QSlider" name="slLength">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>168</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="pageStep">
             <number>24</number>
            </property>
            <property name="value">
             <number>24</number>
            </property>
            <property name="sliderPosition">
             <number>24</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
            <property name="tickPosition">
             <enum>QSlider::NoTicks</enum>
            </property>
            <property name="tickInterval">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spLength">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>168</number>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="lblResolution">
          <property name="text">
           <string>Resolution</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QSlider" name="slResolution">
            <property name="minimum">
             <number>8</number>
            </property>
            <property name="maximum">
             <number>60</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>30</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spResolution">
            <property name="minimum">
             <number>8</number>
            </property>
            <property name="maximum">
             <number>60</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>30</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblGraphType">
          <property name="text">
           <string>Graph type</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cbGraphType">
          <item>
           <property name="text">
            <string>Line</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Stacked</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblScaleType">
          <property name="text">
           <string>Scale type</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="cbScaleType">
          <item>
           <property name="text">
            <string>Standard</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Logarithmic</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lblStyle">
          <property name="text">
           <string>Style</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="cbGraphStyle">
          <item>
           <property name="text">
            <string>RedBlue</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Default</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Blue</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Light</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Clean</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DarkColorized</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DarkGreenBlue</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWebView" name="webView">
      <property name="url">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="log">
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>log</zorder>
   <zorder>webView</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>855</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;Graph</string>
    </property>
    <addaction name="actionExport_as"/>
   </widget>
   <widget class="QMenu" name="menuSnapshots">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;Snapshots</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="actionDelete"/>
    <addaction name="actionClear_all_snapshots"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSnapshots"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGenerate"/>
   <addaction name="actionReset"/>
  </widget>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionSnapshot">
   <property name="text">
    <string>Snapshot</string>
   </property>
  </action>
  <action name="actionCreate">
   <property name="text">
    <string>&amp;Create</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionClear_all_snapshots">
   <property name="text">
    <string>Clear all snapshots</string>
   </property>
  </action>
  <action name="actionGenerate">
   <property name="icon">
    <iconset>
     <normaloff>../refresh.png</normaloff>../refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Create graph with current parameters</string>
   </property>
   <property name="statusTip">
    <string>Create graph with current parameters</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionReset">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Set parameter values to default</string>
   </property>
  </action>
  <action name="actionExport_as">
   <property name="text">
    <string>&amp;Export to...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>chkNow</sender>
   <signal>toggled(bool)</signal>
   <receiver>dtEndDate</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spLength</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slLength</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slLength</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spLength</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spResolution</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slResolution</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>758</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slResolution</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spResolution</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>626</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>758</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbGraphType</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>actionGenerate</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>652</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbScaleType</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>actionGenerate</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>652</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbGraphStyle</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>actionGenerate</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>652</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slLength</sender>
   <signal>sliderReleased()</signal>
   <receiver>actionGenerate</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slResolution</sender>
   <signal>sliderReleased()</signal>
   <receiver>actionGenerate</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>626</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spResolution</sender>
   <signal>editingFinished()</signal>
   <receiver>actionGenerate</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>758</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkNow</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionGenerate</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkIdle</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionGenerate</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkWorking</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionGenerate</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkOffline</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionGenerate</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkPaused</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionGenerate</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dtEndDate</sender>
   <signal>editingFinished()</signal>
   <receiver>actionGenerate</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spLength</sender>
   <signal>editingFinished()</signal>
   <receiver>actionGenerate</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
